e2e/e2etests/e2etests.go

```diff
@@ -472,6 +474,14 @@ var AllE2ETests = []runner.E2ETest{
 		},
 		TestTONDepositAndCall,
 	),
+	runner.NewE2ETest(
+		TestTONDepositRefundName,
```

Comment:
Inconsistency TestTONDepositRefundName <-> TestTONDepositAndCallRefund

I believe it should be `TestTONDepositAndCallRefundName`?